lib/faker/default/gender.rb

```diff
@@ -16,6 +16,19 @@ def type
         fetch('gender.types')
       end
 
+      ##
+      # Produces a gender marker.
+      #
+      # @return [String]
+      #
+      # @example
+      #   Faker::Gender.short_type #=> "x"
+      #
+      # @faker.version 1.9.0
```

Comment:
I'm a first-time contributor as well, but I believe this should be:

```suggestion
      # @faker.version next
```

Reply:
Great catch! Thanks a lot 🙏